llvm/test/DebugInfo/Symbolize/XCOFF/xcoff-symbolize-data.ll

```diff
@@ -0,0 +1,154 @@
+; REQUIRES: powerpc-registered-target
+; RUN: llc -filetype=obj -o %t -mtriple=powerpc-aix-ibm-xcoff < %s
+; RUN: llvm-symbolizer --obj=%t 'DATA 0x60' 'DATA 0x61' 'DATA 0x64' 'DATA 0X68' \
+; RUN:   'DATA 0x90' 'DATA 0x94' 'DATA 0X98' | \
+; RUN:   FileCheck %s
+
+;; This files tests llvm-symbolizer can successfully symbolize the data symbol
+;; from the DWARF info on AIX. Note that AIX is big endian.
```

Comment:
I'd put this comment right at the top of this file.
```suggestion
;; This file shows that llvm-symbolizer can symbolize data symbols
;; from the DWARF info on AIX. Note that AIX is big endian.
```
(Also reflow the comment)

This wasn't quite what I meant with the individual test cases - I'll comment on them directly where appropriate.